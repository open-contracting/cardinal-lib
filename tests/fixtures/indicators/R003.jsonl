{"ocid":"F","tender":{"tenderPeriod":{"startDate":"2020-08-03T18:00:00-05:00","endDate":"2020-08-04T18:00:00-05:00"},"procurementMethod":"open","procurementMethodDetails":"Open"}}
{"ocid":"adequate tender period","tender":{"tenderPeriod":{"startDate":"2020-08-03T18:00:00-05:00","endDate":"2020-08-19T18:00:00-05:00"},"procurementMethod":"open","procurementMethodDetails":"Open"}}
{"ocid":"no tender period start date","tender":{"tenderPeriod":{"startDate":"2020-08-03T18:00:00-05:00"},"procurementMethod":"open","procurementMethodDetails":"Open"}}
{"ocid":"direct procurement method","tender":{"tenderPeriod":{"startDate":"2020-08-03T18:00:00-05:00","endDate":"2020-08-19T18:00:00-05:00"},"procurementMethod":"direct","procurementMethodDetails":"Direct"}}
{"ocid":"no procurement method","tender":{"tenderPeriod":{"startDate":"2020-08-03T18:00:00-05:00","endDate":"2020-08-19T18:00:00-05:00"}}}
