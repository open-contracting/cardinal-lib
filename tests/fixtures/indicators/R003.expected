{"OCID":{"F":{"R003":1.0}}}
